docs/standard/serialization/system-text-json-converters-how-to.md

```diff
@@ -301,6 +326,24 @@ The following code registers the converter:
 
 [!code-csharp[](snippets/system-text-json-how-to/csharp/RoundtripStackOfT.cs?name=SnippetRegister)]
 
+## Handle null values
+
+By default, the serializer does not pass `null` to custom converters for reference types and primitive value types on serialization and deserialization:
```

Comment:
The serializer does pass `null` to value type converters, which normally would throw a JsonException. However, for some converters, like the built-in converter for the value type `JsonElement`, it writes a `null` value if the internal value is `JsonTokenType.Null`.

Also in 5.0 there is now a `HandleNull` public virtual that says whether or not to handle null.

At least this was the behavior in 3.1. cc @layomia ?

Reply:
My summary of default behavior without the HandleNull set to true comes from https://github.com/dotnet/runtime/issues/34439

Reply:
@steveharter is correct and dotnet/runtime#34439 should be updated.

```suggestion
By default, the serializer does not pass `null` to custom converters for reference types on serialization and deserialization. `JsonTokenType.Null` is passed to custom converters for value types on deserialization.
```